deploy/backup/copy-backup.sh

```diff
@@ -35,10 +35,25 @@ get_backup_dest() {
     local backup=$1
 
     if $ctrl get "pxc-backup/$backup" 1>/dev/null 2>/dev/null; then
+        BASE64_DECODE_CMD=""
+        echo $(echo 'abc'|base64) | base64 -d 1>/dev/null 2>/dev/null
+        res1=$?
+        echo $(echo 'abc'|base64) | base64 -D 1>/dev/null 2>/dev/null
+        res2=$?
+
+        if [[ ${res1} == 0 ]]; then
+            BASE64_DECODE_CMD="base64 -d"
+        elif [[ ${res2} == 0 ]]; then
+            BASE64_DECODE_CMD="base64 -D"
+        else
+             echo "base64 decode error."
+             exit 1
+        fi
+
         local secret=$( $ctrl get "pxc-backup/$backup" -o "jsonpath={.status.s3.credentialsSecret}" 2>/dev/null)
         export AWS_ENDPOINT_URL=$(  $ctrl get "pxc-backup/$backup" -o "jsonpath={.status.s3.endpointUrl}" 2>/dev/null)
-        export AWS_ACCESS_KEY_ID=$( $ctrl get "secret/$secret"     -o 'jsonpath={.data.AWS_ACCESS_KEY_ID}'     2>/dev/null | base64 -D)
-        export AWS_SECRET_ACCESS_KEY=$($ctrl get "secret/$secret"  -o 'jsonpath={.data.AWS_SECRET_ACCESS_KEY}' 2>/dev/null | base64 -D)
+        export AWS_ACCESS_KEY_ID=$( $ctrl get "secret/$secret"     -o 'jsonpath={.data.AWS_ACCESS_KEY_ID}'     2>/dev/null | eval ${BASE64_DECODE_CMD})
+        export AWS_SECRET_ACCESS_KEY=$($ctrl get "secret/$secret"  -o 'jsonpath={.data.AWS_SECRET_ACCESS_KEY}' 2>/dev/null | eval ${BASE64_DECODE_CMD})
```

Comment:
@delgod You're right, so I added the command detection code before executing the command, and I passed tests on common systems, pls review the code again.